myhpom/tests/test_cloudfactory_response_view.py

```diff
@@ -73,3 +75,38 @@ def test_successful_review(self):
         # The status and the response should be saved:
         self.assertEqual(CloudFactoryDocumentRun.STATUS_PROCESSED, run.status)
         self.assertEqual(SUCCESS_DATA, run.response_content)
+
+    def test_user_email_task_on_completed_review(self):
+        """
+        When a review has been completed, the view should trigger the email task.
+        """
+        # this seems to be the way to test whether a task has been called...
+        @after_task_publish.connect(sender='myhpom.tasks.EmailUserDocumentReviewCompleted')
+        def task_sent_handler(sender=None, headers=None, body=None, **kwargs):
```

Comment:
You can also do this with a mock - which...if you have other tests that cover the task itself (looks like you do) is a little more succinct:

```
    @patch('myhpom.views.document.EmailUserDocumentReviewCompleted')
    def test_user_email_task_on_completed_review(self, task_mock):
        """
        When a review has been completed, the view should trigger the email task.
        """
        run = CloudFactoryDocumentRun.objects.create(
            run_id='SUCCESS_ID', status=CloudFactoryDocumentRun.STATUS_PROCESSING
        )
        response = self.post_json(reverse('myhpom:cloudfactory_response'), SUCCESS_DATA)
        self.assertEqual(200, response.status_code)  # indicates successful review completion
        task_mock.delay.assert_called_once_with(run.pk, 'http', 'testserver')

```